src/components/button/Button.module.scss

```diff
@@ -39,3 +39,27 @@
     border: 1px solid transparent;
   }
 }
+
+@media only screen and (max-width: $Tablet) {
+  .button {
+    font-size: $CardRecord;
+    width: 240px;
+    padding: indent(4) 0;
+    margin: indent(4) indent(4);
+    border-radius: 30px;
+    border: none;
+  
+    .span {
+      width: 100%;
+    }
+  }
+}
+
+@media only screen and (max-width: $Mobile) {
+  .button {
+    font-size: $Description;
+    width: 170px;
```

Comment:
what about max width for button and flex width? don't fix the width please. if the content changes will you manually change the width? Or will the content pop out of the button?

Reply:
it's definitely make sense
thank you!